### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by chemik-bit**
First docstrings improvement and api-website updates

### CODE REVIEW

File: api-website/index.rst [hunk 1]
```diff
@@ -10,7 +10,20 @@ Welcome to SalesGPT's documentation!
    :maxdepth: 2
    :caption: Contents:
 
+   salesgpt/salesgpt.agents
```
Comment (filip-michalsky) [inline api-website/index.rst:13]: @chemik-bit Can You please make the names more readable (eg `salesgpt.chains` -> Chains etc

**Review by filip-michalsky — APPROVED** [review]
Great update on docstrings! Thank you

**Comment by filip-michalsky** [comment]
Also @chemik-bit please update the website doc compilation README (as well as API readme) to be reproducible - it feels like the README in the website folder is boilerplate and I don't see a README in the API website. Thank you!

**Comment by filip-michalsky** [comment]
@chemik-bit ping me once these small changes are done and then we will merge, ty!

File: api-website/index.rst (hunk 1 above)
Comment (chemik-bit) [reply api-website/index.rst]: Np, done!

**Comment by chemik-bit** [comment]
> Also @chemik-bit please update the website doc compilation README (as well as API readme) to be reproducible - it feels like the README in the website folder is boilerplate and I don't see a README in the API website. Thank you!

@filip-michalsky Not sure about the contents of the README in both the standard and api website? I would simply delete or add to .gitignore as this is simply internal production and deployment guiding.

**Comment by filip-michalsky** [comment]
@chemik-bit is this good to be merged?

**Pull request merged**